client_log.go

```diff
@@ -27,16 +26,10 @@ type LogUploader interface {
 	Upload(api ApiRequester, server string, logs LogData) error
 }
 
-type LogEntry struct {
-	Timestamp string `json:"timestamp"`
-	Level     string `json:"level"`
-	Message   string `json:"message"`
-}
-
 type LogData struct {
 	deviceID     string
 	deploymentID string
-	Messages     []LogEntry `json:"messages"`
+	Messages     []byte `json:"messages"`
```

Comment:
Any reason why we replace a structured log entry that fits what API expects with raw bytes sequence?